react/features/recent-list/functions.any.js

```diff
@@ -20,6 +20,10 @@ export function toDisplayableItem(item, defaultServerURL, t) {
 
     return {
         colorBase: serverName,
+        id: {
```

Comment:
Looking at this, can't we use the `key` just defined below instead? It has the advantage of being already there and being a string, so the comparison is event simpler. WDYT?

Reply:
I don't really want that, because then I need to deconstruct the string later to identify the entry, that can lead to bugs too.

Reply:
Gotcha. I thought we used a map but the entries are in an array so we wouldn't gain much.